samples/canopen/canopen_rpdo/main.cpp

```diff
@@ -106,11 +76,12 @@ int main() {
     timer.stopTimer();
 
     //create the RPDO node
-    TestCanNode testCanNode;
+    RPDOCanNode testCanNode;
 
-    // Reserved memory for CANopen stack usage
-    uint8_t sdoBuffer[1][CO_SDO_BUF_BYTE];
-    CO_TMR_MEM appTmrMem[4];
+    // Reserve memory for the SDO buffers. We take the number of SDOs supported and
```

Comment:
If you're using an acronym you should probably expand it out the first time you use it. It's better for readability and new users.